web/src/locales/de/data.json

```diff
@@ -16,6 +16,7 @@
     "Sync policies successfully": "Sync policies successfully"
   },
   "application": {
+    "Empty": "Empty",
```

Comment:
This is wrong, update i18n as: https://casdoor.org/docs/contributing#how-to-update-i18n-data

Reply:
done